src/main/java/org/example/Main.java

```diff
@@ -50,35 +41,55 @@ public static void main(String[] args) throws Exception {
             JsonObject jsonObject = new Gson().fromJson(response.toString(), JsonObject.class);
             JsonArray results = jsonObject.getAsJsonArray("results");
 
+            Class.forName("org.postgresql.Driver");
+
             Connection jDbCconnection = DriverManager.getConnection("jdbc:postgresql://localhost:5432/dex");
-            PreparedStatement statement = jDbCconnection.prepareStatement("INSERT INTO medication_test (brand_name) VALUES (?)");
+            PreparedStatement statement = jDbCconnection.prepareStatement("INSERT INTO medication_test2 (brand_name, strength, dosage_form) VALUES (?,?,?)");
 
 
 
                 List<String> brandNameList = new ArrayList<>();
                 List<Medication> medications = new ArrayList<>();
                 for (JsonElement resultElement : results) {
                     JsonObject result = resultElement.getAsJsonObject();
-                    JsonArray brandNames = result.getAsJsonObject("openfda").getAsJsonArray("brand_name");
-                    if (brandNames == null) {
+                    JsonArray products = result.getAsJsonArray("products");
+                    String brandName = products.getAsJsonArray().get(0).getAsJsonObject().getAsJsonPrimitive("brand_name").getAsString();
+                    String strength = products.getAsJsonArray().get(0).getAsJsonObject().getAsJsonArray("active_ingredients").getAsJsonArray().get(0).getAsJsonObject().getAsJsonPrimitive("strength").getAsString();
+                    String dosage_form = products.getAsJsonArray().get(0).getAsJsonObject().getAsJsonPrimitive("dosage_form").getAsString();
+                    if (brandName == null) {
                         continue;
                     }
-                    for (JsonElement brandNameElement : brandNames) {
-                        String brandName = brandNameElement.getAsString();
-
-                        Medication medication = new Medication();
-                        medication.setBrand_name(brandName);
-                        medications.add(medication);
-
-                        statement.setString(1, brandName);
-                        statement.execute();
-                    }
+                    brandNameList.add(brandName);
+
+                    Medication medication = new Medication();
+                    medication.setBrand_name(brandName);
+                    medication.setStrength(strength);
+                    medication.setDosage_form(dosage_form);
+                    medications.add(medication);
+
+                    statement.setString(1, brandName);
+                    statement.setString(2, strength);
+                    statement.setString(3, dosage_form);
+                    statement.execute();
+//                    for (JsonElement brandNameElement : brandName) {
```

Comment:
statement is handling the order of my sql query here, for every iteration it'll execute this row i've defined further up in the code.